### ISSUE TITLE
FMoW?

### ISSUE THREAD

**Issue by Nyquixt** [Aaditya-Singh/Low-Shot-Robustness#6]
Hi authors,

Amazing work! I did some linear probing on FMoW with CLIP-RN50 and got a test accuracy around 39% (which isn't great) so I would expect the few-shot fine-tuning would yield a lower accuracy. But any particular reason why FMoW was not included in the preprint? I think the distribution gap between FMoW and pre-training data (natural images) is somewhat similar to the case of Camelyon17. Happy to hear!

-K

**Comment by Aaditya-Singh** [comment]
Hi K,

Thanks for your interest in our work! [The LP-FT paper](https://arxiv.org/pdf/2202.10054.pdf) includes experiment on FMoW and Table 2 also shows 37% OOD accuracy with CLIP RN50. I'd recommend trying out CLIP ViTB as it performed substantially better than RN50 in our experiments (see Table 9), but it's quite possible that the performance will be much lower on the low-shot subsets. 

You're right in pointing out that FMoW's a relevant dataset which consists of both natural images and natural distribution shift, but as per the [WILDS benchmark](https://wilds.stanford.edu/datasets/) there is also a subpopulation shift in the dataset. That would've introduced another confounding factor, and thoroughly studying such kinds of shifts was beyond the scope of our study.

Regards,
Aaditya
